modules/basegl/src/datavisualizer/volumeraycastvisualizer.cpp

```diff
@@ -95,17 +96,17 @@ std::vector<Processor*> VolumeRaycastVisualizer::addVisualizerNetwork(Outport* o
     auto vbb = net->addProcessor(util::makeProcessor<VolumeBoundingBox>(GP{8, 3}));
     auto lrp = net->addProcessor(util::makeProcessor<LineRendererProcessor>(GP{8, 6}));
 
-    static_cast<FloatVec4Property*>(bak->getPropertyByIdentifier("bgColor1"))
+    dynamic_cast<FloatVec4Property*>(bak->getPropertyByIdentifier("bgColor1"))
```

Comment:
No reason to dynamic cast if not checking for null afterwards?

Reply:
a dynamic cast will fail with a crash, a static cast will just corrupt data and go on...

Reply:
i.e. it is much easier do debug a dynamic cast failing